<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-Q9TFL0ME" ignored="0" name="Default test" tests="1" failures="1" timestamp="28 Jan 2022 18:46:20 GMT" time="20.698" errors="0">
  <testcase name="Inbound_NM_CustomerEmailToTwilio" time="12.457" classname="com.twilioTests.TwilioEmailTest">
    <failure type="com.twilio.exception.ApiException" message="Authenticate">
      <![CDATA[com.twilio.exception.ApiException: Authenticate
at com.twilio.rest.api.v2010.account.MessageReader.pageForRequest(MessageReader.java:243)
at com.twilio.rest.api.v2010.account.MessageReader.firstPage(MessageReader.java:170)
at com.twilio.rest.api.v2010.account.MessageReader.read(MessageReader.java:150)
at com.twilio.base.Reader.read(Reader.java:24)
at com.pages.LoginTwilloPage.getMessages(LoginTwilloPage.java:88)
at com.pages.LoginTwilloPage.getMessage(LoginTwilloPage.java:82)
at com.pages.LoginTwilloPage.login(LoginTwilloPage.java:58)
at com.twilioTests.TwilioEmailTest.Inbound_NM_CustomerEmailToTwilio(TwilioEmailTest.java:67)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- Inbound_NM_CustomerEmailToTwilio -->
</testsuite> <!-- Default test -->
